Common/src/main/java/com/blamejared/crafttweaker/api/ingredient/IngredientCacheBuster.java

```diff
@@ -0,0 +1,55 @@
+package com.blamejared.crafttweaker.api.ingredient;
+
+import com.blamejared.crafttweaker.mixin.common.access.item.AccessIngredient;
+import com.blamejared.crafttweaker.platform.Services;
+import net.minecraft.world.item.crafting.Ingredient;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Used to bust the {@link Ingredient}#itemStacks cache when the {@link Ingredient} was resolved during a Server reload (When an instance of {@link net.minecraft.tags.TagManager} is available.
+ */
+public class IngredientCacheBuster {
+    
+    private static boolean claimed;
```

Comment:
No thread safety: yay or nay? Maybe if we'd need it, what about a thread local instance?

Reply:
I don't think we need it, the class will be moved to `impl` instead of `api`.

If anyone needs to use the class for their own script load, please get in contact with us and we can see what we can do.